build.sh

```diff
@@ -10,14 +10,20 @@ mkdir docs/dist
 
 ./clean-cmake.sh
 ARGON_JS_BUILD_BUILD_WITH_SIMD=1 ./build-wasm.sh
-cp dist/argon2.wasm docs/dist/argon2-simd.wasm
-mv dist/argon2.wasm dist/argon2-simd.wasm
+# cp dist/argon2.wasm docs/dist/argon2-simd.wasm
+# mv dist/argon2.wasm dist/argon2-simd.wasm
 
 ./clean-cmake.sh
 ARGON_JS_BUILD_BUILD_WITH_SIMD=0 ./build-wasm.sh
-cp dist/argon2.js docs/dist/argon2.js
-cp dist/argon2.wasm docs/dist/argon2.wasm
+# cp dist/argon2.js docs/dist/argon2.js
+# cp dist/argon2.wasm docs/dist/argon2.wasm
```

Comment:
Just making sure these are meant to be left commented out!

Reply:
Yessir!